django_projects/locallibrary/catalog/models.py

```diff
@@ -89,3 +88,7 @@ def get_absolute_url(self):
     def __str__(self):
         """String for representing the Model object."""
         return f'{self.last_name}, {self.first_name}'
+
+    @property
```

Comment:
ngoài cách dùng decorator `@property` thì còn cách nào khác để lấy data books theo author sử dụng ở template ko em?
decorator `@property` chỉ nên dùng để tính toán từ các field của model thôi nha

Reply:
ngoài cách sử dụng @property thì có thể dùng function base view đúng không ạ

Reply:
em xóa phần xử lý ở model sử dụng decorator `@property` đi nha
Phần này sẽ xử lý trong class tương ứng ở views, trong TH này là class `AuthorDetailView`, override method `get_context_data()`. Em tìm hiểu rồi apply